packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -13,7 +13,24 @@ import 'colors.dart';
 const double _kTabBarHeight = 50.0;
 
 const Color _kDefaultTabBarBackgroundColor = const Color(0xCCF8F8F8);
-
+const Color _kDefaultTabBarBorderColor = const Color(0x4C000000);
+
+/// An iOS styled bottom navigation tab bar.
+///
+/// Displays multiple tabs using [BottomNavigationBarItem] with one tab being
+/// active, the first tab by default.
+///
+/// This [StatelessWidget] doesn't store the active tab itself. You must
+/// listen to the [onTap] callbacks and call `setState` with a new [currentIndex]
+/// for the new selection to reflect.
+///
+/// Tab changes typically trigger a switch between [Navigator]s, each with its
+/// own navigation stack, per standard iOS design.
+///
+/// If the given [backgroundColor] is translucent, it will produce a blurring
```

Comment:
the color can only be semi-transparent, not translucent. I would just say "If the given [backgroundColor] has an opacity less than 1.0..."